.github/workflows/Downgrade.yml

```diff
@@ -10,20 +10,32 @@ on:
       - master
     paths-ignore:
       - 'docs/**'
+  schedule:
+    - cron: '28 12 * * 6'
```

Comment:
What's this do?

Reply:
It runs the tests once per week, where the time is chosen randomly per repo, such that not all repos trigger on the same time.

Reply:
👍